src/components/Task/Task.tsx

```diff
@@ -0,0 +1,39 @@
+import { forwardRef, type ParamHTMLAttributes } from 'react';
```

Comment:
I think I brought this up in another PR of yours.   Same comment. We shouldn't utilize forwardRef untless there's deliberate reason for its use not just beause it might be convinient later.